ci/indy-pool.dockerfile

```diff
@@ -29,7 +29,7 @@ ARG indy_node_ver=1.9.0~dev1009
 ARG python3_indy_crypto_ver=0.4.5
 ARG indy_crypto_ver=0.4.5
 
-RUN apt-get update -y && apt-get install -y \
+RUN apt-get update -y && apt-get install -y --allow-unauthenticated \
```

Comment:
do we need the allow-unauthenticated parameter?

Reply:
SF's keys were updated then `indy-pool.dockerfile` in master was updated. So allow-unauthenticated can be removed from the PR after merge with master